lib/Conversion/TorchToTcp/Utils.cpp

```diff
@@ -62,6 +62,64 @@ Value broadcastRankInLeadingDims(ConversionPatternRewriter &rewriter,
       input.getDefiningOp()->getLoc(), resultType, input, reassociationMap);
 }
 
+Value broadcastRank0Dor1DToND(ConversionPatternRewriter &rewriter, Value input,
+                              int64_t targetRank, int64_t axisInOutput) {
+  RankedTensorType inputType = input.getType().cast<RankedTensorType>();
+  auto inputRank = inputType.getRank();
+  assert(inputRank < 2 && "Only 0D and 1D tensors are supported!");
```

Comment:
This assert feels like it should be accompanied by a match failure (or op->emitError) in the matchAndRewrite function calling this. I don't see that at first glance.

Reply:
Good point. Added that check in the caller.